worlds/subnautica/docs/setup_en.md

```diff
@@ -2,7 +2,7 @@
 
 ## Required Software
 
-- Subnautica from: [Subnautica Steam Store Page](https://store.steampowered.com/app/264710/Subnautica/)
+- Subnautica from: [Subnautica Steam Store Page](https://store.steampowered.com/app/264710/Subnautica/) | [Subnautica Epic Game Store Page](https://store.epicgames.com/en/p/subnautica)
```

Comment:
https://store.epicgames.com/p/subnautica
```suggestion
- Subnautica from: [Subnautica Steam Store Page](https://store.steampowered.com/app/264710/Subnautica/) | [Subnautica Epic Game Store Page](https://store.epicgames.com/p/subnautica)
```

This link will infer the user language from browser info, rather than setting it to english.